primitives/pipelined.futil

```diff
@@ -1,12 +1,12 @@
 extern "pipelined.sv" {
     // A latency-sensitive multiplier that takes 4 cycles to compute its result.
-    static<4> primitive pipelined_mult (
+    static<4> primitive pipelined_mult[WIDTH] (
```

Comment:
According to the new static interface (#1725), where `static<n>` states that the II of the module is `n`, this should be `static<1>` right? CC @calebmkim @paili0628

Reply:
Are we ready to use the newest iterations of the static interface @andrewb1999? That might be another layer of complication if we can't support it. 

I'm not up to date on what #1725 entails, but wouldn't both II and latency need to be tracked by a static component as two separate attributes?

Reply:
I saw the comments about static referring to the II before but doesn't that prevent us from using pipelined primitives in a non-pipelined context? I almost think that latency and II should be two different parts of the static interface.

Reply:
Yeah, on the current plan, we would like to make this `static<1>`. However, you can still use them it a non-pipelined context (i.e., read the output of the multiplier after 4 cycles): you'll just have to know that yourself (i.e., you'll just have to know to read the outputs after four cycles, that information won't be exposed in the interface anywhere). 

The the latency annotation will tell you the II. Maybe in the future it'll become a big enough problem where we'll expose two numbers: one for the latency and one for the II.

Reply:
> The the latency annotation will tell you the II.

Do you mean the `static` annotation here? If so, yeah that makes. The latency annotation should be a future addition and kind of approximates the kind of reasoning that Filament does.